.github/workflows/deep-sea-stories-deploy.yml

```
@@ -0,0 +1,33 @@
+name: Deploy deep-sea-stories
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  build-deploy:
+    if: github.actor != 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup SSH
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan -H ${{ secrets.SSH_HOST }} >> ~/.ssh/known_hosts 2>/dev/null
```

This defeats the purpose of adding the key to known hosts.
Instead, this should be something like
```suggestion
          echo "${{ vars.KNOWN_HOSTS }}" >> ~/.ssh/known_hosts
```
and `KNOWN_HOSTS` should be the expected line in known_hosts
```
<host> <public-key>
```